addons/website_slides/models/slide_slide_resource.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, fields, models
+
+
+class SlideResource(models.Model):
+    _name = 'slide.slide.resource'
+    _description = "Additional resource for a particular slide"
+
+    slide_id = fields.Many2one('slide.slide', required=True, ondelete='cascade')
+    type = fields.Selection([('file', 'File'), ('url', 'Link')], required=True, default="url")
+    name = fields.Char('Name', required=True)
+    data = fields.Binary('Resource', compute='_reset_resources', store=True)
+    link = fields.Char('Link', compute='_reset_resources', store=True)
```

Comment:
Should you add `readonly=False` ?

Reply:
I do not see why I should explicitly say the fields are `not readonly` since this attribute is set to `False` by default.

Reply:
Because on computed field, the default value for "readonly" is True

Reply:
In this case, it makes sense. Resolving it.